x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/html_strip.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { FieldNameField } from './common_fields/field_name_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import { TargetField } from './common_fields/target_field';
+
+export const HtmlStrip: FunctionComponent = () => {
+  return (
+    <>
+      <FieldNameField
+        helpText={i18n.translate(
+          'xpack.ingestPipelines.pipelineEditor.htmlStripForm.fieldNameHelpText',
+          { defaultMessage: 'The string-valued field to remove HTML tags from.' }
+        )}
+      />
+
+      <TargetField
+        helpText={i18n.translate(
+          'xpack.ingestPipelines.pipelineEditor.htmlStripForm.targetFieldHelpText',
+          {
+            defaultMessage:
+              'The field to assign the stripped value to. If blank, the field will be updated in-place.',
```

Comment:
Can you replace the `{{original field}}` reference here?
```suggestion
              'Field used to contain stripped text. Defaults to {{original field}}.',
```